CMakeLists.txt

```diff
@@ -386,15 +386,16 @@ option(BUILD_SHARED_LIBS "Build package with shared libraries." OFF)
 #################################
 
 # This will build a static library using all specified modules
-option(DRISHTI_BUILD_WORLD "Build single library" ON)
+option(DRISHTI_INSTALL "Run package-config installation" ON)
```

Comment:
What is the point of this option? If user don't want to install this library then he will not call `cmake --build _builds --target install`. Is this for 3rd parties when Drishti added by `add_subdirectory(3rdParty/Drishti)`?

Reply:
Yes, that was the intention.  I'm not currently testing in this mode, but I've added such flags to other libs in the past for this reason.   Probably inverting this option (i.e., `DRISHTI_SKIP_INSTALL`) is a little clearer for such a use case.  We can rename or remove.